### PR TITLE
Remove cmdline argument --logfie, fixes #79

### LINKED ISSUE #79: using latest geth version breaks wait_for_geth_to_start
the reason is the --logfile flag is not understood :
```

➜  testpopulus git:(master) ✗ nice -n 20 geth --genesis /home/lotso/PycharmProjects/testpopulus/chains/default/genesis-block.json --datadir /home/lotso/PycharmProjects/testpopulus/chains/default --maxpeers 0 --networkid 123456 --port 42700 --ipcpath /tmp/tmpReFq6q --shh --rpcapi db,eth,net,web3,shh --minerthreads 1 --logfile /home/lotso/PycharmProjects/testpopulus/chains/default/logs/deploy-dry-run-20160411-160948.log --unlock 0 --password /usr/local/lib/python2.7/dist-packages/populus/default_blockchain_password --nodiscover --mine --rpc
Incorrect Usage.

NAME:
   geth - the go-ethereum command line interface

USAGE:
   geth [options] command [command options] [arguments...]

VERSION:
   1.4.0-unstable

COMMANDS:
   import       import a blockchain file
   export       export blockchain into file
   upgradedb    upgrade chainblock database
   removedb     Remove blockchain and state databases
   dump         dump a specific block from storage
   monitor      Geth Monitor: node metrics monitoring and visualization
   makedag      generate ethash dag (for testing)
   gpuinfo      gpuinfo
   gpubench     benchmark GPU
   version      print ethereum version numbers
   wallet       ethereum presale wallet
   account      manage accounts
   init         bootstraps and initialises a new genesis block (JSON)
   console      Geth Console: interactive JavaScript environment
   attach       Geth Console: interactive JavaScript environment (connect to node)
   js           executes the given JavaScript files in the Geth JavaScript VM
   help, h      Shows a list of commands or help for one command

ETHEREUM OPTIONS:
  --datadir "/home/lotso/.ethereum"     Data directory for the databases and keystore
  --keystore                            Directory for the keystore (default = inside the datadir)
  --networkid "1"                       Network identifier (integer, 0=Olympic, 1=Frontier, 2=Morden)
  --olympic                             Olympic network: pre-configured pre-release test network
  --testnet                             Morden network: pre-configured test network with modified starting nonces (replay protection)
  --dev                                 Developer mode: pre-configured private network with several debugging flags
  --genesis                             Insert/overwrite the genesis block (JSON format)
  --identity                            Custom node name
  --fast                                Enable fast syncing through state downloads
  --lightkdf                            Reduce key-derivation RAM & CPU usage at some expense of KDF strength
  --cache "128"                         Megabytes of memory allocated to internal caching (min 16MB / database forced)
  --blockchainversion "3"               Blockchain version (integer)

ACCOUNT OPTIONS:
  --unlock      Comma separated list of accounts to unlock
  --password    Password file to use for non-inteactive password input

API AND CONSOLE OPTIONS:
  --rpc                                                         Enable the HTTP-RPC server
  --rpcaddr "localhost"                                         HTTP-RPC server listening interface
  --rpcport "8545"                                              HTTP-RPC server listening port
  --rpcapi "eth,net,web3"                                       API's offered over the HTTP-RPC interface
  --ws                                                          Enable the WS-RPC server
  --wsaddr "localhost"                                          WS-RPC server listening interface
  --wsport "8546"                                               WS-RPC server listening port
  --wsapi "eth,net,web3"                                        API's offered over the WS-RPC interface
  --wsdomains                                                   Domains from which to accept websockets requests (can be spoofed)
  --ipcdisable                                                  Disable the IPC-RPC server
  --ipcapi "admin,eth,debug,miner,net,shh,txpool,personal,web3" API's offered over the IPC-RPC interface
  --ipcpath "geth.ipc"                                          Filename for IPC socket/pipe within the datadir (explicit paths escape it)
  --rpccorsdomain                                               Comma separated list of domains from which to accept cross origin requests (browser enforced)
  --jspath "."                                                  JavaScript root path for `loadScript` and document root for `admin.httpGet`
  --exec                                                        Execute JavaScript statement (only in combination with console/attach)

NETWORKING OPTIONS:
  --bootnodes           Comma separated enode URLs for P2P discovery bootstrap
  --port "30303"        Network listening port
  --maxpeers "25"       Maximum number of network peers (network disabled if set to 0)
  --maxpendpeers "0"    Maximum number of pending connection attempts (defaults used if set to 0)
  --nat "any"           NAT port mapping mechanism (any|none|upnp|pmp|extip:<IP>)
  --nodiscover          Disables the peer discovery mechanism (manual peer addition)
  --nodekey             P2P node key file
  --nodekeyhex          P2P node key as hex (for testing)

MINER OPTIONS:
  --mine                        Enable mining
  --minerthreads "4"            Number of CPU threads to use for mining
  --minergpus                   List of GPUs to use for mining (e.g. '0,1' will use the first two GPUs found)
  --autodag                     Enable automatic DAG pregeneration
  --etherbase "0"               Public address for block mining rewards (default = first account created)
  --targetgaslimit "4712388"    Target gas limit sets the artificial target gas floor for the blocks to mine
  --gasprice "20000000000"      Minimal gas price to accept for mining a transactions
  --extradata                   Block extra data set by the miner (default = client version)

GAS PRICE ORACLE OPTIONS:
  --gpomin "20000000000"        Minimum suggested gas price
  --gpomax "500000000000"       Maximum suggested gas price
  --gpofull "80"                Full block threshold for gas price calculation (%)
  --gpobasedown "10"            Suggested gas price base step down ratio (1/1000)
  --gpobaseup "100"             Suggested gas price base step up ratio (1/1000)
  --gpobasecf "110"             Suggested gas price base correction factor (%)

VIRTUAL MACHINE OPTIONS:
  --jitvm               Enable the JIT VM
  --forcejit            Force the JIT VM to take precedence
  --jitcache "64"       Amount of cached JIT VM programs

LOGGING AND DEBUGGING OPTIONS:
  --metrics                     Enable metrics collection and reporting
  --verbosity "3"               Logging verbosity: 0=silent, 1=error, 2=warn, 3=info, 4=core, 5=debug, 6=detail
  --vmodule                     Per-module verbosity: comma-separated list of <pattern>=<level> (e.g. eth/*=6,p2p=5)
  --backtrace ":0"              Request a stack trace at a specific logging statement (e.g. "block.go:271")
  --pprof                       Enable the pprof HTTP server
  --pprofport "6060"            pprof HTTP server listening port
  --memprofilerate "524288"     Turn on memory profiling with the given rate
  --blockprofilerate "0"        Turn on block profiling with the given rate
  --cpuprofile                  Write CPU profile to the given file
  --trace                       Write execution trace to the given file

EXPERIMENTAL OPTIONS:
  --shh         Enable Whisper
  --natspec     Enable NatSpec confirmation notice

MISCELLANEOUS OPTIONS:
  --solc "solc" Solidity compiler command to be used
  --help, -h    show help


flag provided but not defined: -logfile
```

```

➜  testpopulous git:(master) ✗ populus deploy -d
Exception in thread Thread-2:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/threading.py", line 754, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/local/lib/python2.7/dist-packages/populus/geth.py", line 78, in enqueue_stream
    logfile.write(line)
ValueError: I/O operation on closed file

Traceback (most recent call last):
  File "/usr/local/bin/populus", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/populus/cli/deploy_cmd.py", line 116, in deploy
    wait_for_geth_to_start(dry_run_proc)
  File "/usr/local/lib/python2.7/dist-packages/populus/geth.py", line 408, in wait_for_geth_to_start
    raise ValueError("Geth process never started\n\n{0}".format(''.join(output)))
ValueError: Geth process never started
```

### PR DESCRIPTION
**Pull request by holiman**
Fix #79

### CODE REVIEW

File: populus/geth.py [hunk 1]
```diff
@@ -181,10 +181,12 @@ def geth_wrapper(data_dir, popen_class=subprocess.Popen, cmd="geth",
...
-    if logfile is not None:
-        command.extend((
-            '--logfile', logfile,
-        ))
+    # This parameter is deprecated, 
+    #see https://github.com/pipermerriam/populus/issues/79
+    #if logfile is not None:
+    #    command.extend((
+    #        '--logfile', logfile,
+    #    ))
```
Comment (pipermerriam) [inline populus/geth.py]: I think I'd rather just have this code block deleted.

**Comment by holiman** [comment]
ping

**Comment by pipermerriam** [comment]
@holiman thanks for the ping.  Looks like tests failed because of a network error, rerunning them and I'll get this merged and released when it's green.

**Comment by pipermerriam** [comment]
closing in favor of #86 so I can fix thi CI issues

**Comment by pipermerriam** [comment]
New release available as `0.8.0` on pypi.

**Pull request closed without merging**